s32/drivers/s32ze/BaseNXP/header/S32Z2.h

```diff
@@ -1,237 +1,238 @@
-#include "S32Z2_NETC_F3.h"
-#include "S32Z2_NETC_F3_COMMON.h"
-#include "S32Z2_NETC_F3_GLOBAL.h"
-#include "S32Z2_NETC_F3_PCI_HDR_TYPE0.h"
-#include "S32Z2_NETC_F3_SI0.h"
-#include "S32Z2_NETC_F3_SI1.h"
-#include "S32Z2_NETC_F3_SI2.h"
-#include "S32Z2_NETC_F3_SI3.h"
-#include "S32Z2_NETC_F3_SI4.h"
-#include "S32Z2_NETC_F3_SI5.h"
-#include "S32Z2_NETC_F3_SI6.h"
-#include "S32Z2_NETC_F3_SI7.h"
-#include "S32Z2_NETC_IERB.h"
-#include "S32Z2_NETC_PRIV.h"
-#include "S32Z2_NETC_VF1_PCI_HDR_TYPE0.h"
-#include "S32Z2_NETC_VF2_PCI_HDR_TYPE0.h"
-#include "S32Z2_NETC_VF3_PCI_HDR_TYPE0.h"
-#include "S32Z2_NETC_VF4_PCI_HDR_TYPE0.h"
-#include "S32Z2_NETC_VF5_PCI_HDR_TYPE0.h"
-#include "S32Z2_NETC_VF6_PCI_HDR_TYPE0.h"
-#include "S32Z2_NETC_VF7_PCI_HDR_TYPE0.h"
-#include "S32Z2_OCOTP.h"
-#include "S32Z2_NVIC.h"
-#include "S32Z2_SCB.h"
-#include "S32Z2_OMU.h"
-#include "S32Z2_PIT.h"
-#include "S32Z2_PLLDIG.h"
-#include "S32Z2_PMC.h"
-#include "S32Z2_PMSS_SAFETY.h"
-#include "S32Z2_POWER_SCALING_UNIT.h"
-#include "S32Z2_PROFILER.h"
-#include "S32Z2_PROGRAM_MEMORY_SUBSYSTEM.h"
-#include "S32Z2_PSI5.h"
-#include "S32Z2_PSI5_S.h"
-#include "S32Z2_QMAN_CNTRL.h"
-#include "S32Z2_QUADSPI.h"
-#include "S32Z2_QUADSPI_ARDB.h"
-#include "S32Z2_QUEUE_DESCRIPTOR.h"
-#include "S32Z2_QUEUE_MANAGER.h"
-#include "S32Z2_QUEUE_MANAGER1.h"
-#include "S32Z2_QUEUE_MANAGER2.h"
-#include "S32Z2_QUEUE_MANAGER3.h"
-#include "S32Z2_RDC.h"
-#include "S32Z2_RESULT_DMACRC.h"
-#include "S32Z2_RESULT_DMA_MP.h"
-#include "S32Z2_RESULT_DMA_TCD.h"
-#include "S32Z2_ROUND_ROBIN_ARBITER.h"
-#include "S32Z2_RTT.h"
-#include "S32Z2_RTUE_NIC_D.h"
-#include "S32Z2_RTUF_NIC_D.h"
-#include "S32Z2_RTUM_NIC_D.h"
-#include "S32Z2_RTUP_NIC_B.h"
-#include "S32Z2_RTU_GPR.h"
-#include "S32Z2_RTU_L_VFCCU.h"
-#include "S32Z2_RTU_MC_CGM.h"
-#include "S32Z2_RTU_MRU.h"
-#include "S32Z2_RTU_PMC.h"
-#include "S32Z2_RTU_SEMA42.h"
-#include "S32Z2_RTU_XRDC.h"
-#include "S32Z2_RXLUT.h"
-#include "S32Z2_SBSW.h"
-#include "S32Z2_SDA_AP.h"
-#include "S32Z2_SEC_S250.h"
-#include "S32Z2_SEMA42.h"
-#include "S32Z2_SINC.h"
-#include "S32Z2_SIPI.h"
-#include "S32Z2_SMU_L_VFCCU.h"
-#include "S32Z2_SMU_MRU.h"
-#include "S32Z2_SMU_SEMA42.h"
-#include "S32Z2_SMU_SRG_S.h"
-#include "S32Z2_SMU_XRDC.h"
-#include "S32Z2_SPFU.h"
-#include "S32Z2_SPI.h"
-#include "S32Z2_SRAMCTL.h"
-#include "S32Z2_SRX.h"
-#include "S32Z2_SW_ETH_MAC_PORT0.h"
-#include "S32Z2_SW_ETH_MAC_PORT1.h"
-#include "S32Z2_SW_PORT0.h"
-#include "S32Z2_SW_PORT1.h"
-#include "S32Z2_SW_PORT2.h"
-#include "S32Z2_SW_PSEUDO_MAC_PORT2.h"
-#include "S32Z2_TIMERS.h"
-#include "S32Z2_TMR0_BASE.h"
-#include "S32Z2_TMU.h"
-#include "S32Z2_TRGMUX_0.h"
-#include "S32Z2_TRGMUX_1.h"
-#include "S32Z2_TRGMUX_2.h"
-#include "S32Z2_TRGMUX_3.h"
-#include "S32Z2_USDHC.h"
-#include "S32Z2_VIRT_WRAP.h"
-#include "S32Z2_WATCHDOG.h"
-#include "S32Z2_XBIC.h"
-#include "S32Z2_XRDC.h"
-#include "S32Z2_SYSTICK.h"
-#include "S32Z2_MPU.h"
-
-#endif  /* #if !defined(S32Z2_H_) */
+/*

```

Comment:
What is happening with the line ending here, pls make sure all files have LF line break type

Reply:
I updated the EOL